target/linux/ath79/dts/qca9533_mikrotik_routerboard-mapl-2nd.dts

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca9533_mikrotik_routerboard-16m.dtsi"
+
+/ {
+	compatible = "mikrotik,routerboard-mapl-2nd", "qca,qca9533";
+	model = "MikroTik RouterBOARD mAPL-2nD (mAP lite)";
```

Comment:
The model string should match the compatible string:
`mikrotik,routerboard-mapl-2nd-map-lite`

Reply:
@adschm Or is it okay?

Reply:
"mAP lite" is an _alternate_ name, and it would not make sense to include two names into the compatible. So, IMO the current choice is okay. Whether one should choose mAPL-2nD or "mAP lite" for the primary name is subject of ongoing disagreement, though.